modules/http/src/main/java/com/spotify/ffwd/http/HttpPing.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2013-2017 Spotify AB. All rights reserved.
+ *
+ * The contents of this file are licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.spotify.ffwd.http;
+
+import com.netflix.loadbalancer.IPing;
+import com.netflix.loadbalancer.Server;
+import javax.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class HttpPing implements IPing {
+    @Inject
+    HttpClientFactory clientFactory;
+
+    @Override
+    public boolean isAlive(final Server server) {
+        try {
+            clientFactory.newClient(server).ping().get();
+        } catch (Exception e) {
+            log.warn("Error when pinging server ({}): {}", server, e.getMessage());
```

Comment:
logging same msg at 2 diff log levels

Reply:
They are subtly different. One will only include the message of the error (`warn`), the other will include the stacktrace (`trace`). This exception is expected to happen fairly often, so logging the stacktrace by default would be spammy.

Reply:
Kind of a hacky solution, but it works.